packages/core/src/components/sections/ProductShelf/Overrides.tsx

```diff
@@ -0,0 +1,10 @@
+import ProductCard from 'src/components/product/ProductCard'
+
+import ProductShelfCustomizations from 'src/customizations/components/overrides/ProductShelf'
+
+const Components = {
+    ProductCard,
+    ...ProductShelfCustomizations.components
+}
+
+export { Components }
```

Comment:
Makes GH happy.
<img width="243" alt="Screenshot 2023-04-06 at 11 14 50" src="https://user-images.githubusercontent.com/11325562/230404465-5ff6ac31-4807-447f-84a1-3932cefdb23b.png">

```suggestion
export { Components }

```